glue/sales_val_flagging.py

```diff
@@ -193,7 +232,18 @@ def group_size_adjustment(df, stat_groups: list, min_threshold, condos: bool):
         lambda row: "_".join([str(row[col]) for col in stat_groups]), axis=1
     )
 
-    return df_flagged_updated
+    df = df_flagged_updated.assign(
+        # PTAX-203 binary
+        sv_is_ptax_outlier=lambda df: df["sv_ind_ptax_flag_w_deviation"],
+        na=False,
+        sv_is_heuristic_outlier=lambda df: (
+            ~df["sv_outlier_reason1"].str.contains("PTAX-203", na=False)
+        )
```

Comment:
**[Nitpick, non-blocking]** This expression relies on the assumption that PTAX-203 flags always get highest priority in the list of reasons. Perhaps a more robust solution would just be to check if `df["sv_in_ptax_flag_w_deviation"]` is not 1?